sp800-63-3/sec5_xAL.md

```diff
@@ -110,9 +112,11 @@ Separating IAL and AAL also works for traditional M-04-04 LOA use cases, such as
 
 #### 5.3.1. Business Process vs. Online Transaction
 
-An online transaction may not be equivalent to a complete business process that requires offline processing, or online processing in a completely segmented system. In selecting the appropriate assurance levels, the agency should assess the risk associated with online transactions they are offering via the digital service, not the entire business process associated with the provided benefit or service. For example, in an online survey, sensitive personally identifiable information may be collected, but it is never made available online to the person after the information is submitted. In this instance, it is important for the information to be carefully protected in backend systems, but there is no reason to identity proof or even authenticate the user providing the information. The online transaction is solely a submission of the data only. The entire business process may require a significant amount of data validation, without ever needing to know if the correct person submitted the correct information regarding their family. In this scenario, there is no need for any xAL.
+An online transaction may not be equivalent to a complete business process that requires offline processing, or online processing in a completely separate system. The agency should assess the risk of the portion of a business process available via the digital service, not the entire business process that is executed outside the digital service to provide the benefit or service. For example, in an online survey, sensitive information may be collected, but it is never made available online after the information is submitted. In this instance, it is important for the information to be carefully protected in backend systems, but there is no reason to identity proof or even authenticate the user providing the information. The online transaction is solely a submission of the data only. The entire business process may require a significant amount of data validation, but to submit the data there is no need proof or authenticate.
+
+Another example is a digital service that accepts resumes to apply for open job postings. In this case, the service allows any individual to submit a resume on behalf of someone else, and in subsequent visits to the site, access the resume. Since the resume contains personal information and is provided in later sessions, the agency must select an AAL2 authenticator, at a minimum.
```

Comment:
Agencies don't select the authenticator, they select the AAL. So "agency must select an AAL2 authenticator" -> "agency must select at least AAL2 authentication" (is it "must" or "shall"?)